functions_lab.js

```diff
@@ -0,0 +1,309 @@
+// # Function Exercises
+//
+// ## Question 1
+//
+// Write a function that returns the largest of two numbers. If
+// they are equal, return the second one.
+//
+// Example:
+//
+// Input: `12, 4`
+//
+// Output: `12`
+
+function largerNum (param1, param2) {
```

Comment:
Since the question asked for returning value, we should use return instead of console.log inside this function. 
A better code would be something like this: 
function largerNum (param1, param2) {
  if (param1 > param2) {
    return param1
  } else if (param1 < param2) {
    return param2
  }
}